charts/ceph-csi-cephfs/README.md

```diff
@@ -164,6 +164,8 @@ charts and their default values.
 | `provisioner.podSecurityPolicy.enabled`        | Specifies whether podSecurityPolicy is enabled                                                                                                       | `false`                                            |
 | `provisionerSocketFile`                        | The filename of the provisioner socket                                                                                                               | `csi-provisioner.sock`                             |
 | `pluginSocketFile`                             | The filename of the plugin socket                                                                                                                    | `csi.sock`                                         |
+| `readAffinity.enabled` | Enable read affinity for RBD volumes. Recommended to set to true if running kernel 5.8 or newer. | `false` |
```

Comment:
CephFS subvolumes

Reply:
Do we need kernel 5.8 for cephfs?

Reply:
`read_from_replica` option is from kernel 5.8
https://docs.ceph.com/en/latest/man/8/rbd/#kernel-rbd-krbd-options
I couldn't find any other doc.